src/controllers/userController.js

```diff
@@ -326,24 +338,24 @@ class userController {
      * @memberof userController
      */
   static async updateProfile(req, res) {
+    const { body, user } = req;
+    const { userID } = user;
+    const {
+      firstName,
+      lastName,
+      gender,
+      birthDate,
+      preferedLang,
+      preferedCurrency,
+      residence,
+      department,
+      managerEmail,
+      imageUrl,
+      bio,
+      passportNumber
+    } = body;
     try {
-      const { body, user } = req;
-      const { userID } = user;
-      const {
-        firstName,
-        lastName,
-        gender,
-        birthDate,
-        preferedLang,
-        preferedCurrency,
-        residence,
-        department,
-        managerEmail,
-        imageUrl,
-        bio,
-        passportNumber
-      } = body;
-      const result = await Models.Users.update(
+      await Models.Users.update(
```

Comment:
please wait for my PR to be merged for I have created a migration that adds isUpdated attribute to the user's table
 and in the end, update isUpdated to true
```suggestion
  const isUpdated = true,
   await Models.Users.update({
   ...
   isUpdated,
   })
```

Reply:
@blaise82  Sure thing brother, you can see that my branch is not yet deployed because of that ``` isUpdated ``` attribute which is missing.